wled00/data/settings_sync.htm

```diff
@@ -44,7 +44,77 @@ <h3>Realtime</h3>
 Start universe: <input name="EU" type="number" min="0" max="63999" required><br>
 <i>Reboot required.</i> Check out <a href="https://github.com/LedFx/LedFx" target="_blank">LedFx</a>!<br>
 Skip out-of-sequence packets: <input type="checkbox" name="ES"><br>
-DMX start address: <input name="DA" type="number" min="0" max="510" required><br>
+<div>
```

Comment:
This could be done dynamically in a similar way as buses are created.

Reply:
i agree, but i am not great at html+js and i couldn't figure out all the internal code paths that result in the right dynamic behavior to add/remove/serialize etc.